src/utils/triangle.ts

```diff
@@ -0,0 +1,15 @@
+const getTrianglePerimetr = (a: number, b: number, c: number): number =>
+  a + b + c;
+
+export const getTriangleArea = (a: number, b: number, c: number): number => {
```

Comment:
[CODE STYLE] - The `getTriangleArea` function should not accept any parameters. According to the checklist, all necessary values should be available as object fields. If this function is meant to be a method of a class that contains `a`, `b`, and `c` as fields, it should use `this.a`, `this.b`, and `this.c` instead of taking parameters.